lib/src/migrators/module.dart

```diff
@@ -776,7 +791,12 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
         }
       }
       if (ruleUrl != null) {
-        if (_useAllowed) {
+        if (builtInOnly) {
+          _upstreamStylesheets.add(currentUrl);
+          if (migrateDependencies) visitDependency(ruleUrl, import.span);
+          _upstreamStylesheets.remove(currentUrl);
+          return;
```

Comment:
Why return eagerly here? What if there are multiple imports in a single `@import` that all need to be traversed?

Reply:
Good catch. Fixed